fastlane/metadata/ar-SA/keywords.txt

```diff
@@ -1 +1 @@
-اجتماعي,ملاحظات,jetpack,كتابة,وسم جغرافي,وسائط,موقع ويب,موقع إلكتروني,تدوين,مجلة
\ No newline at end of file
+مدون، كتابة، تدوين، ويب، صانع، عبر الإنترنت، متجر، أعمال، إعداد، إنشاء، كتابة، مدونات
```

Comment:
This is less than the required 100 characters ✅

But I'm not sure about "jetpack" having been translated. I think the same rationale applies as WordPress...

I'll fix it at the source later, too.